host-bmc/host_pdr_handler.cpp

```diff
@@ -767,10 +766,10 @@ void HostPDRHandler::processHostPDRs(mctp_eid_t /*eid*/,
 
         for (const auto& [terminusHandle, terminusInfo] : tlPDRInfo)
         {
-            info("TerminusHandle: {TH}", "TH", terminusHandle);
-            info("TID: {TID}", "TID", std::get<0>(terminusInfo));
-            info("EID: {EID}", "EID", std::get<1>(terminusInfo));
-            info("Validity: {VALID}", "VALID", std::get<2>(terminusInfo));
+            info(
+                "TerminusHandle:'{TH}', TID:'{TID}', EID:'{EID}', Validity:{VALID}",
```

Comment:
give meaningful name 
what is TH?  make it HANDLE so while checking lg2:logs it can be readable HANDLE: value

Reply:
The name is TerminusHandle itself right? TH is the variable that holds the value.

Reply:
when someone looks lg2 structure object where difficult to identify what TH means, but if you keep TERMINAL_HANDLE then it's you are improve readability of the logs.
e.g. **printing lg2 structure** :
     PRIORITY=6
    MESSAGE=Time mode has been changed to xyz.openbmc_project.Time.Synchronization.Method.NTP
    LOG2_FMTMSG=Time mode has been changed to {MODE}
    CODE_FILE=/usr/src/debug/phosphor-time-manager/1.0+git/manager.cpp
    CODE_LINE=98
    CODE_FUNC=bool phosphor::time::Manager::setCurrentTimeMode(const std::string&)
    MODE=xyz.openbmc_project.Time.Synchronization.Method.NTP

Reply:
okay done!